packages/env-tests/src/scaffold.ts

```diff
@@ -1,11 +1,19 @@
 import { concurrentMap } from '@celo/base'
+import { CeloContract } from '@celo/contractkit'
+import { newStableToken } from '@celo/contractkit/lib/generated/StableToken'
+import { StableTokenWrapper } from '@celo/contractkit/lib/wrappers/StableTokenWrapper'
 import { generateKeys } from '@celo/utils/lib/account'
 import { privateKeyToAddress } from '@celo/utils/lib/address'
 import BigNumber from 'bignumber.js'
 import { EnvTestContext } from './context'
 
 BigNumber.config({ EXPONENTIAL_AT: 1e9 })
 
+export const StableTokenToRegistryName: Record<string, string> = {
+  CUSD: 'StableToken',
+  CEUR: 'StableTokenEur',
+}
```

Comment:
```suggestion
export const StableTokenToRegistryName: Record<string, CeloContract> = {
  CUSD: CeloContract.StableToken,
  CEUR: "StableTokenEUR" as CeloContract,
}
```

Until we can just do `CeloContract.StableTokenEUR`